pkg/operator/controllers/pullsecret/pullsecret_controller.go

```diff
@@ -184,15 +184,22 @@ func (r *Reconciler) ensureGlobalPullSecret(ctx context.Context, operatorSecret,
 		// allows for simpler logic flow, when delete and create are not handled separately
 		// this call happens only when there is a need to change, it has no significant impact on performance
 		err := r.client.Delete(ctx, secret)
+		r.log.Info("Global Pull secret Not Found, Creating Again")
```

Comment:
It should be one line above or after `if err` clause.
It would be better that there is always a `if err` clause just after it gets `err`.